Sources/Commands/SwiftPackageTool.swift

```diff
@@ -314,11 +323,15 @@ public class SwiftPackageTool: SwiftTool<PackageToolOptions> {
                 $0.outputPath = $3?.path
             })
         binder.bind(
-            option: generateXcodeParser.add(
+            generateXcodeParser.add(
                 option: "--legacy-scheme-generator", kind: Bool.self,
                 usage: "Use the legacy scheme generator"),
+            generateXcodeParser.add(
+                option: "--watch", kind: Bool.self,
+                usage: "Watch the filesystem and autogenerate the Xcode project if needed"),
```

Comment:
Would it be worth being more specific?

`Watch for changes to the Package manifest to regenerate the Xcode project`